attendance/templates/attendance/user_page.html

```diff
@@ -40,7 +40,20 @@ <h3 class="panel-title">
   <div class="panel-body">
     <ul class="collabsible-list list-group " id="users-at-work">
     {% for user in at_work_users %}
-			<li><i class="fa fa-circle-o" aria-hidden="true"></i> {{user.first_name}} {{user.last_name}}</li>
+            <li>
+			  <table><tr><td>
+              <i class="fa fa-circle-o" aria-hidden="true"></i> {{user.first_name}} {{user.last_name}}
+              {% if request.user.is_staff %}

```

Comment:
@DavidKozina 

Here only staff will see the button. Also superuser has to see the button. 

This should be also validated in `view` so direct POST request can't logout user.